conda_smithy/ci_skeleton.py

```diff
@@ -6,21 +6,20 @@
 added to conda-forge's queue.
 """
 
-import os
+from pathlib import Path
 import sys
 
 from .configure_feedstock import make_jinja_env
 
 
 def _render_template(template_file, env, forge_dir, config):
     """Renders the template"""
-    template = env.get_template(
-        os.path.basename(template_file) + ".ci-skel.tmpl"
-    )
-    target_fname = os.path.join(forge_dir, template_file)
-    print("Generating " + target_fname, file=sys.stderr)
+    template_file_name = str(Path(template_file).name)
```

Comment:
I don't think you need the call to `str()` here; `Path(...).name` is already a `str` object.